### ISSUE TITLE
about reimplement in val set and test set

### ISSUE THREAD

**Issue by linqinghong** [speedinghzl/CCNet#26]
hi~
i using the project in your pytorch-segmentation-toolbox.
when i training the pspnet in the project.
i get the result follow:
1) val:0.7883(ss) / 0.7930(ms) | test:0.7756(ss) / 0.7846(ms)
2) val:0.7897(ss) / 0.8003(ms) | test:0.7800(ss) / 0.7826(ms)
so i want to make sure it is normal? because it seem like we can easily get a good result in val set, but have some trouble in test set.

i want to know how's your performance in val and test set in the same experiment.(pspnet as well as ccnet)~thanks for your reply!

**Comment by speedinghzl** [comment]
almost the same.
